src/App.js

```diff
@@ -83,6 +86,28 @@ function App() {
     });
   }
 
+  const handleModalOpen = (item, itemId) => {
+    // set item to be deleted to item object and set itemId since it's separate from Firestore
+    setItemToDelete({ ...item, id: itemId });
+    setShowModal(true);
```

Comment:
This is an interesting approach to save the item object for deletion in App state before doing so.

Reply:
This is more of a question for clarity on my end, but why is the ID "separate from Firestore"? It seems we need to associate the id with an item in the Firestore in order to delete it, but it is not coming from the item object itself?

Reply:
@Cjstoney The structure of the data returned from Firestore has the document id "separate" from the field data (`doc.id` has the id and `doc.data()` returns an object with the other fields and values). We originally didn't do anything to combine these values and just used item and itemId to hold these different bits from Firestore. 

This sprint Nick and I decided to reorganize the data returned from Firestore before passing it down through the app, so that we'd have a simpler, cleaner array of items with item.id and some other helpful properties we needed. We coordinated with Connie & Sheila about this coming change, and that may have informed this line if I'm remembering correctly, since `setItemToDelete({...item, id: itemId})` would work with their current code but be really easy to simplify to `setItemToDelete(item)` once we merge our branches together and no longer have a separate itemId prop.